js_task1/secondTask.js

```diff
@@ -0,0 +1,120 @@
+const generateButton = document.querySelector(".generate-btn");
+const checkButton = document.querySelector(".check-btn");
+
+const elementsCount = 6; // if we need moore domino, or less
+const minValue = 1;
+const maxValue = 3;
+
+let firstArray = [];
+let secondArray = [];
+
+function randomValue(min, max) {
+  //get random value in range
+  min = Math.ceil(min);
+  max = Math.floor(max);
+
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+function randomArray(length) {
+  //get random array
+  let arr = [];
+
+  for (let i = 0; i < length; i++) {
+    arr.push(randomValue(minValue, maxValue));
+  }
+
+  return arr;
+}
+
+function isFullRow(array) {
```

Comment:
Is this method used somewhere? If not please remove it

Reply:
done